src/main/java/org/jenkinsci/plugins/zap/ZAPDriver.java

```diff
@@ -557,6 +557,10 @@ private void checkParams(AbstractBuild<?, ?> build, BuildListener listener) thro
         if (this.evaluatedTargetURL == null || this.evaluatedTargetURL.isEmpty()) throw new IllegalArgumentException("STARTING POINT (URL) IS MISSING, PROVIDED [ " + this.evaluatedTargetURL + " ]");
         else Utils.loggerMessage(listener, 1, "(EXP) STARTING POINT (URL) = [ {0} ]", this.evaluatedTargetURL);
 
+        this.evaluatedLoginURL = envVars.expand(this.evaluatedLoginURL);
+        if (this.evaluatedLoginURL == null || this.evaluatedLoginURL.isEmpty()) throw new IllegalArgumentException("LOGIN FORM TARGET (URL) IS MISSING, PROVIDED [ " + this.evaluatedLoginURL + " ]");
```

Comment:
This validation should be performed only if `authMode` is true, otherwise it requires a login URL even if no authentication is enabled.